### PR TITLE
chore: update nsx product terms

### PR DESCRIPTION
**Pull request by tenthirtyam**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/blob/main/CONTRIBUTING.md) for making a pull request.

**Summary of Pull Request**

- Updated all instance of "NSX-T Data Center" to "NSX" product name.
- Updated all instance of "NSX Management Cluster" to "NSX Manager cluster" term.
- Updated one instance of "NSX Local Manager" to "NSX Manager" term.
- Updated module version from v2.3.0.1001 to v2.3.0.1002.

No `CHANGELOG.md` update is necessary.

**Type of Pull Request**

- [ ] This is a bug fix.
- [ ] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [ ] This is a refactoring update.
- [x] This is something else.
      Please describe: `chore`

**Context of the Pull Request***

Update to product name and terms.

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

- [ ] Tests have been completed (for bug fixes / features).
- [x] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: PowerValidatedSolutions.psm1 [hunk 1]
```diff
@@ -18647,7 +18647,7 @@ Export-ModuleMember -Function Set-WsaPasswordPolicy
 #####################################################################
 
 #####################################################################
-#Region     Begin NSX-T Data Center Functions                  ######
+#Region     Begin NSX Functions                  ######
```
Comment (GaryJBlake) [inline PowerValidatedSolutions.psm1:18650]: Needs spaces adding back so hashes are aligned at the end

File: PowerValidatedSolutions.psm1 [hunk 2]
```diff
@@ -22159,7 +22159,7 @@ Function Set-NsxtApplianceUserPassword {
 }
 Export-ModuleMember -Function Set-NsxtApplianceUserPassword
 
-#EndRegion  End NSX-T Functions                                ######
+#EndRegion  End NSX Functions                                ######
```
Comment (GaryJBlake) [inline PowerValidatedSolutions.psm1:22162]: Needs spaces added to align hashes

**Review by GaryJBlake — CHANGES_REQUESTED** [review]

File: PowerValidatedSolutions.psm1 (hunk 2 above)
Comment (tenthirtyam) [reply PowerValidatedSolutions.psm1:22162]: Addressed in https://github.com/vmware-samples/power-validated-solutions-for-cloud-foundation/pull/252/commits/b858c50d0a5573c5284a93d4384b3db0d4ece27c.

**Review by GaryJBlake — APPROVED** [review]
LGTM!

**Pull request merged**